packages/generators/generators/lib/plops/__tests__/.gitignore

```diff
@@ -0,0 +1 @@
+output
```

Comment:
Actually we don't need this anymore as I delete the folder after each test 🤔
But I guess we could still add a `__test__/output` to the package gitignore in case the cleanup doesn't work at some point and we end up committing that folder.